Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityOptimizeWindow.cs

```diff
@@ -0,0 +1,607 @@
+                                                Undo.RecordObject(this.discoveredMaterials[i], "Convert to MRTK Standard shader");
+                                                ConvertMaterial(this.discoveredMaterials[i]);
+                                            }
+                                        EditorGUILayout.EndHorizontal();
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            GUILayout.EndVertical();
+            EditorGUILayout.Space();
+        }
+
+        private void RenderSceneOptimizations()
+        {
+            GUILayout.BeginVertical("Box");
+            EditorGUILayout.LabelField(this.ToolbarTitles[(int)ToolbarSection.Scene], BoldLargeTitle);
+            using (new EditorGUI.IndentLevelScope())
+            {
+                EditorGUILayout.LabelField("This section provides controls and performance information for the currently opened scene. Any optimizations performed are only for the active scene at any moment. Also be aware that changes made while in editor Play mode will not be saved.", EditorStyles.wordWrappedLabel);
+
+                BuildSection("Lighting Settings", "https://docs.unity3d.com/Manual/GlobalIllumination.html", () =>
+                {
+                    EditorGUILayout.BeginHorizontal();
+                        EditorGUILayout.LabelField("Global Illumination can produce great visual results but sometimes at great expense.", EditorStyles.wordWrappedLabel);
+                        if (GUILayout.Button(new GUIContent("View Lighting Settings", "Open Lighting Settings"), EditorStyles.miniButton, GUILayout.Width(160f)))
+                        {
+                            EditorApplication.ExecuteMenuItem("Window/Rendering/Lighting Settings");
+                        }
+                    EditorGUILayout.EndHorizontal();
+
+                    disableRealtimeGlobalIllumination = EditorGUILayout.ToggleLeft("Disable Realtime Global Illumination", disableRealtimeGlobalIllumination);
+
+                    if (this.PerfTarget == PerformanceTarget.AR_Headsets)
+                    {
+                        disableBakedGlobalIllumination = EditorGUILayout.ToggleLeft("Disable Baked Global Illumination", disableBakedGlobalIllumination);
+                    }
+
+                    if (GUILayout.Button("Optimize Lighting"))
+                    {
+                        OptimizeScene();
+                    }
+                });
+
+                BuildSection("Live Scene Analysis", null, () =>
+                {
+                    EditorGUILayout.BeginHorizontal();
+                    EditorGUILayout.LabelField(lastAnalyzedTime == null ? "Click analysis button for MRTK to scan your currently opened scene" : "Scanned " + GetRelativeTime(lastAnalyzedTime));
+
+                    if (GUILayout.Button("Analyze Scene", GUILayout.Width(200f)))
```

Comment:
Super nit: This button is over-sized compared to other similar buttons (eg View Lighting Settings)